django_json_ld/util.py

```diff
@@ -3,6 +3,17 @@
 from django.core.serializers.json import DjangoJSONEncoder
 
 
+def validate_sd(sd):
+    if sd and type(sd) != dict:
```

Comment:
```suggestion
    if sd and not isinstance(sd, collections.Mapping):
```
Using `isinstance` allows subclasses of `dict` to pass validation.

I've used `collections.Mapping` since " It works for `dict()`, `collections.OrderedDict()`, and `collections.UserDict()`". See: https://stackoverflow.com/questions/25231989/how-to-check-if-a-variable-is-a-dictionary-in-python#comment89139687_25232010

Note that this requires a `import collections` at the top.